docs/userguide/text.md

```diff
@@ -1,144 +1,151 @@
+- `bold`: Make the text bold.
+- `itelic`: Make the text italic.
+- `variant_numeric`: Specifies the style of rendering numbers. Allowed values are `"normal"`,
+        `"ordinal"`,
+        `"slashed-zero"`,
+        `"lining-nums"`,
+        `"oldstyle-nums"`,
+        `"proportinal-nums"`,
+        `"tabular-nums"`,
+        `"diagonal-fractions"` or
+        `"stacked-fractions"`.
+See [SVG documentation](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/font-variant)
+for its meaning.
+
+Each parameter may also be `None` (the default value). This means that the value will be inherited
+from its parent style (see below).
+
+If you want to font or text properties that are not available in `TextStyle`, please
+[let us know](https://github.com/spirali/elsie/issues/new).
+
+You can pass a text style as a second argument to the `text` method:
+```elsie,width=400,height=150
+slide.text("Hello world!", elsie.TextStyle(size=70, color="red"))
+```
 
+*Tip*: to avoid repeating the `TextStyle` class name, you can import it via some short alias:
 ```python
-# Setting style globally
-elsie.set_style("big_red", elsie.TextStyle(size=70, color="red"))
-
-@elsie.slide()
-def text1(slide):
-    slide.text("Hello world!", "big_red")
+from elsie import TextStyle as s
+...
+slide.box().text("Hello world!", s(color="red"))
 ```
 
+### Text style inheritance
+Each box also inherits a list of named styles from its parent. If you use a string instead of a
+`TextStyle`, *Elsie* will try to find a style with the given name. The two following examples
+produce the same slide as the example above:
+```elsie,width=400,height=150
+# Setting a style globally for all slides
+slides.set_style("big_red", elsie.TextStyle(size=70, color="red"))
+slide = slides.new_slide(name="slide1")
+slide.text("Hello world!", "big_red")
+```
 
-```python
-@elsie.slide()
-def text1(slide):
-    box = slide.box()
-
-    # Setting style locally for a box
-    box.set_style("big_red", elsie.TextStyle(size=70, color="red"))
+```elsie,width=400,height=150
+box = slide.box()
 
-    box.text("Hello world!", "big_red")
+# Setting style locally for a box
+box.set_style("big_red", elsie.TextStyle(size=70, color="red"))
+box.text("Hello world!", "big_red")
 ```
 
-There are several predifined global styles.
+### Predefined text styles
+There are several predefined global styles available by default. You can use them by their name:
 
-* *"default"* - ``TextStyle(font="Ubuntu", color="black", size=28, line_spacing=1.20, align="middle", variant_numeric="lining-nums")``
-* *"tt"* - ``TextStyle(font="Ubuntu mono")``
-* *"emph"* - ``TextStyle(italic=True)``
-* *"alert"* - ``TextStyle(bold=True, color="red")``
-* *"code"* - ``TextStyle(font="Ubuntu Mono", align="left", color="#222" line_spacing=1.20, size=20)``
-* *"code_lineno"* - ``TextStyle(color="gray")``
+- *"default"* - `TextStyle(font="Ubuntu", color="black", size=28, line_spacing=1.20, align="middle", variant_numeric="lining-nums")`
```

Comment:
Ubuntu font is no longer default font (I will update the docs later)

Reply:
I updated it according to the code.